components/PopupAssignee.vue

```diff
@@ -55,13 +64,13 @@ export default {
   },
   watch: {
     localAssignees: {
-      handler(newValue) {
+      handler (newValue) {
```

Comment:
```suggestion
      handler(newValue) {
```

Reply:
Can you check this? I don't know is it right or wrong but I think there is no space between function name and bracket

Reply:
This is opinionated from code base to code base. The Eslint is configured that way.